vtkext/private/module/Testing/TestF3DEXRMemReader.cxx

```diff
@@ -0,0 +1,57 @@
+#include <vtkImageData.h>
+#include <vtkNew.h>
+
+#include "vtkF3DEXRReader.h"
+
+#include <fstream>
+#include <iostream>
+#include <vector>
+/**
+ * image file taken from
+ * https://github.com/AcademySoftwareFoundation/openexr/blob/370db2835843ac75f85e1386c05455f26a6ff58c/website/test_images/Chromaticities/Rec709.rst
```

Comment:
im not sure how that .rst file is related to the .exr file ?

Reply:
the rst file is just a markdown file on github showing the exr file and it details. I just grabbed it because it was smaller than the other exr files in the repo which made the unit test run a but faster.

https://github.com/AcademySoftwareFoundation/openexr/blob/370db2835843ac75f85e1386c05455f26a6ff58c/website/test_images/Chromaticities/Rec709.rst

Reply:
I think it would be better to just use an existing one. `grayscale.exr` for example is very small.

Reply:
indeed, better to use an existing one.

Reply:
you can remove this line, since you list it in data licenses now.

Reply:
please remove this line